clock_exercise/DeleteButton.qml

```diff
@@ -0,0 +1,51 @@
+import QtQuick 2.0
+
+Rectangle {
+    id: button
```

Comment:
Ok, questo a livello di naming è un il caso diametralmente opposto a `LineUnderTitleOfThePage`: in quel caso la componente era generica ma il nome invece era super-specifico.
Stavolta invece la componente è super-specifica (questo tasto di delete serve a richiedere la cancellazione, ma solo se ci sono allarmi selezionati) ma il nome è generico, lasciando presupporre che io possa usare questo tasto di delete altrove, salvo poi rendermi conto che per motivi ignoti è invisibile se in un'altra pagina dell'applicativo non ho selezionato degli allarmi attraverso il modello C++.